charts/rancher-cis-benchmark/0.2.0/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.9-alpha1
```

Comment:
Should be 2.6.0-alpha1, this is going into 2.6. Anything in 2.5.x line would be 0.1.3 in the dev-v2.5. branch.